docs/guides/other-guides/testbed.md

```diff
@@ -234,6 +234,18 @@ This section describes step by step how to deploy the OSISM Testbed.
 
    You can then use the monitoring services without having to install a complete Openstack environment.
 
+   You can install some basic openstack resources like public network, flavors and images by running
+
+   ```sh
+   /opt/configuration/scripts/bootstrap.sh
+   ```
+
+   If you want to verify the installation with `refstack` run
```

Comment:
I think it makes sense to link https://opendev.org/openinfra/refstack here instead of `refstack`